package.json

```diff
@@ -3,9 +3,9 @@
   "description": "Cockpit isn't a Node package, these are devel time deps, not needed to build tarball either",
   "private": true,
   "dependencies": {
-    "@patternfly/patternfly": "4.80.3",
-    "@patternfly/react-console": "4.2.18",
-    "@patternfly/react-core": "4.91.0",
+    "@patternfly/patternfly": "^4.80.3",
```

Comment:
Ugh, this is a bug in npm-update. It left this `^` in the file. @KKoukiou, are you interested in looking into that? I'm happy to do it, as I most probably broke it with the grouping stuff; just wanted to ask.

Reply:
I am happy to check it